app/core/logger.ts

```diff
@@ -55,13 +55,23 @@ export class Logger {
 
         global && this.global();
 
-        if ( store ) this.store();
+        if ( store || this._loadLevel() ) this.store();
 
     }
 
-    private _loadLevel = (): Level => Number(localStorage.getItem( this._storeAs ));
-
-    private _storeLevel(level: Level) { localStorage[ this._storeAs ] = level; }
+    private _loadLevel(): Level {
+        if(typeof localStorage === 'undefined'){
```

Comment:
I believe returning undefined would be better, that way we can differentiate between no value at all and the user specifically configuring it as OFF.